### PR TITLE
Checking members tests

### PR DESCRIPTION
**Pull request by raonismaneoto**

### CODE REVIEW

File: src/test/java/cloud/fogbow/rcs/core/intercomponent/xmpp/requesters/RemoteGetAllServicesRequestTest.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
+package cloud.fogbow.rcs.core.intercomponent.xmpp.requesters;
+
+import cloud.fogbow.common.util.GsonHolder;
+import cloud.fogbow.rcs.core.BaseUnitTests;
+import cloud.fogbow.rcs.core.TestUtils;
+import cloud.fogbow.rcs.core.intercomponent.RemoteFacade;
```
Comment (wassoares) [inline src/test/java/cloud/fogbow/rcs/core/intercomponent/xmpp/requesters/RemoteGetAllServicesRequestTest.java]: Remove unused import.

File: src/test/java/cloud/fogbow/rcs/core/service/CatalogServiceTest.java [hunk 2]
```diff
@@ -234,4 +240,55 @@ public void testGetLocalMember() {
...
+    @Test
+    public void testGetRemoteCatalogFrom() throws Exception {
+        //setup
+        List<String> members = new ArrayList<>();
+        members.add(TestUtils.FAKE_MEMBER_SERVICE_KEY);
+        Mockito.doReturn(members).when(service).requestMembers();
+
+        RemoteGetAllServicesRequest.Builder requestBuilder = Mockito.spy(RemoteGetAllServicesRequest.builder());
+        PowerMockito.mockStatic(RemoteGetAllServicesRequest.class);
+        PowerMockito.doReturn(requestBuilder).when(RemoteGetAllServicesRequest.class, "builder");
```
Comment (wassoares) [inline src/test/java/cloud/fogbow/rcs/core/service/CatalogServiceTest.java]: Migrate this string to a constant.

**Review by wassoares — APPROVED** [review]
Good job @raonismaneoto... I've commented on two minor corrections you can make, but I've already left your PR approved.

**Review by pauloewerton — APPROVED** [review]
lgtm

**Pull request merged**